PWG/EMCAL/EMCALtasks/AliEmcalCorrectionCellEnergy.cxx

```diff
@@ -135,59 +135,58 @@ Int_t AliEmcalCorrectionCellEnergy::InitRecalib()
   
   Int_t runRC = fEventManager.InputEvent()->GetRunNumber();
   
-  AliOADBContainer *contRF=new AliOADBContainer("");
+  std::unique_ptr<AliOADBContainer> contRF;
+  std::unique_ptr<TFile> fRecalib;
```

Comment:
I know this isn't your fault (I don't think we caught this when we inherited this code), but since we're modifying this now and making the scope slightly wider, can you change the names from `fRecalib` to `recalibFile` or something else? Right now, this looks like a member (it confused me for a bit)

Reply:
Yes, this comes from you! It is fine with me to change it but you could have fixed it yourself before.

Reply:
Well, it came from someone before us, but in any case, I didn't notice it until now. I'm also happy to fix it myself - I just asked because this is the first time that I noticed it and it seemed like it would be straightforward given that we are modifying it now

Reply:
Hi @raymondEhlers ,

since I am anyhow working on it let's clean the variable names right away in this commit! I will take care of it.

Cheers

Markus

Reply:
Thanks!